src/hubverse_infrastructure/hubs/iam.py

```diff
@@ -17,24 +15,24 @@ def create_trust_policy(org: str, repo: str):
     github_policy_document = aws.iam.get_policy_document(
         statements=[
             aws.iam.GetPolicyDocumentStatementArgs(
-                actions=["sts:AssumeRoleWithWebIdentity"],
+                actions=['sts:AssumeRoleWithWebIdentity'],
```

Comment:
Usage of single quotes vs double quotes should be standardized across the hubverse. Other packages, like hubEnsembles, have fully passing linting checks that uses double quotes, so I'd suggest setting that as the standard.

Reply:
Thanks, @lshandross--just pushed a commit to fix this up!